action/object.go

```diff
@@ -0,0 +1,35 @@
+package action
+
+import (
+	log "github.com/sirupsen/logrus"
+
+	"github.com/yunify/qsctl/constants"
+	"github.com/yunify/qsctl/contexts"
+	"github.com/yunify/qsctl/helper"
+)
+
+// DeleteObject will delete a remote object.
+func DeleteObject(remote string) (err error) {
+	bucketName, objectKey, err := ParseQsPath(remote)
+	if err != nil {
+		return
+	}
+
+	if objectKey == "" {
+		return constants.ErrorQsPathObjectKeyRequired
+	}
+
+	_, err = contexts.SetupBuckets(bucketName, "")
+	if err != nil {
+		return
+	}
+	// Head to check whether object not found or forbidden
+	if _, err = helper.HeadObject(objectKey); err != nil {
```

Comment:
We should handle normal error and print user-friendly messages like:

404 -> object key not found
403 -> not enough permission

Reply:
Roger that